backends/p4tools/modules/smith/scripts/compilation-test.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 
-set -e  # Exit on error.
+set -e # Exit on error.
+
+# List of known bugs.
+KNOWN_BUGS=(
+    "h.eth_hdr.src_addr \* 36646024123162; not implemented"
```

Comment:
These error patterns are now too specific. Try to catch the most specific, generic error message. For example. "Div or Mux not implemented"